docker/docker-compose-without-neo4j.override.yml

```diff
@@ -28,11 +28,13 @@ services:
     env_file: datahub-gms/env/docker-without-neo4j.env
     environment:
       - DATAHUB_SERVER_TYPE=${DATAHUB_SERVER_TYPE:-quickstart}
-      - DATAHUB_TELEMETRY_ENABLED=${DATAHUB_TELEMETRY_ENABLED:-true}
+      - DATAHUB_TELEMETRY_ENABLED=${DATAHUB_TELEMETRY_ENABLED}
     depends_on:
       - mysql
     volumes:
       - ${HOME}/.datahub/plugins:/etc/datahub/plugins
+      - ${HOME}/.datahub/plugins/auth/resources/:/etc/datahub/plugins/auth/resources
```

Comment:
@mohdsiddique why do we need this additional line, given that we are mounting the plugins directory already?
cc @jjoyce0510